### DISCUSSION TITLE
Is it possible to update the electron version beyond 13.0 to the newer ones?

### DISCUSSION THREAD

**Discussion by sbesic-starke** [nklayman/vue-cli-plugin-electron-builder#1824] [Q&A] [answered]
There's up to stable version 20.0 now so I am curios, I've seen a lot of bug fixes, stabilizations and features added after 13 that would be nice to have.

**Comment by MatthijsBurgh** [comment, marked as answer]
Yes, but you have to do it manually. And it might require other dependencies to be updated as well. Which probably requires code changes.

I have updated my own project to electron 17. To get a hint of the required changes see the history of my project, https://github.com/tue-robotics/hero-dashboard
